content/en/network_performance_monitoring/network_page.md

```diff
@@ -0,0 +1,125 @@
+3. Click `Create`.
+
+Once the custom facet is created, use this tag to filter and aggregate traffic in the network page and map. All custom facets can be viewed in the bottom `Custom` section of the facet panels.
+
+## Network Data
+
+{{< img src="network_performance_monitoring/network_page/network_data.png" alt="network data"  style="width:90%;" >}}
+
+Your network metrics are displayed through the graphs and the associated table. All sent and received metrics are displayed from the perspective of the source :
+
+* **Sent metrics**: measure the value of something from the _source_ to the _destination_ from the source's perspective.
+* **Received metrics**: measure the value of something from the _destination_ to the _source_ from the source's perspective.
+
+Values displayed might be different for `sent_metric(source to destination)` and `received_metric(destination to source)` if there is a large number of packet drops. In this case, if the `destination` sends a lot of bytes to the `source`, the flows that originate at `destination` include those bytes, but the flows that originate at `source` do not see them as received.
+
+**Note**: The default collection interval is five minutes and retention is seven days.
+
+### Metrics
+
+#### Network Load
+
+The following network load metrics are available:
+
+| Metric | Description |
+| -------- | ------ |
+| **Volume** | The number of bytes sent or received over a period. Measured in bytes (or orders of magnitude thereof) bidirectional.|
+| **Throughput** | The rate of bytes sent or received over a period. Measured in bytes per second, bidirectional. |
+| **Retransmits** | Retransmits represent detected failures that are retransmitted to ensure delivery. Measured in count of retransmits from the `source`. |
+
+#### TCP
+
+TCP is a connection-oriented protocol that guarantees in-order delivery of packets. The following TCP metrics are available:
+
+| Metric | Description |
+| -------- | ------ |
+| **Retransmits** | Retransmits represent detected failures that are retransmitted to ensure delivery. Measured in count of retransmits from the `source`. |
+| **Round-trip Time (RTT)** | Round-trip time is a proxy for latency.|
+| **RTT Variance** | RTT is a proxy for jitter. |
+
+### DNS Resolution
+
+Starting with Agent 7.17+, using reverse-DNS lookup, the Agent resolves IP’s to human-readable domain names for external and internal traffic. DNS allows you to monitor cloud provider endpoints where a Datadog Agent cannot be installed, such as S3 buckets, application load balancers, and API’s. Unrecognizable domain names such as DGA domains from C&C servers may point to network security threats. **DNS is encoded as a tag in Datadog**, so you can use it in search bar queries and the facet panel to aggregate and filter traffic.
+
+{{< img src="network_performance_monitoring/network_page/dns_aggregation.png" alt="DNS aggregation" >}}
+
+**Note**: DNS resolution is supported only for agents running on a host.
```

Comment:
'Agents running in host mode' is a bit different than 'agents running on a host.' We can either replace this with 'agents running in host mode' of if you think it's clearer, we can change this to 'DNS resolution is currently supported for hosts where the system probe is running on the root network namespace, which is usually caused by running the system-probe in a container without using the host network.'